com.unity.multiplayer.mlapi/Runtime/Core/RPCQueue/RPCQueueProcessing.cs

```diff
@@ -25,7 +25,7 @@ internal class RPCQueueProcessing
 #endif
         // Batcher object used to manage the RPC batching on the send side
         private MessageBatcher batcher = new MessageBatcher();
-        private int BatchThreshold = 1000;
+        private int BatchThreshold = 512;
```

Comment:
`const`?

Reply:
Well, it probably should be exposed as a property field...but since this will get "jobified" it might be ok for now.